src/main/java/pl/edu/mimuw/chess/Plansza.java

```diff
@@ -0,0 +1,160 @@
+package pl.edu.mimuw.chess;
+
+import pl.edu.mimuw.Figura;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Plansza {
+  public static final ArrayList<Figura> biale = new ArrayList<>();
```

Comment:
pansza powinna miec dorobione metody do dostawania się do pol z mapy, a nie tak jak jest to robione na rzywca przez `get` w poszczegolnych miejscach i wszedzie jakby tak samo